src/main/java/at/hannibal2/skyhanni/config/commands/Commands.kt

```diff
@@ -216,11 +216,13 @@ object Commands {
         ) { FarmingWeightDisplay.lookUpCommand(it) }
         registerCommand(
             "shcopytranslation",
-            "<language code (2 letters)> <messsage to translate>\n" +
-                "Requires the Chat > Translator feature to be enabled.\n" +
-                "Copies the translation for a given message to your clipboard. " +
-                "Language codes are at the end of the translation when you click on a message."
+            "Translates a message in English to another language via clipboard.\n" +
```

Comment:
"a message in english" can be replaced to " an English message"

Reply:
I prefer the first one